shell.nix

```
@@ -0,0 +1,10 @@
+{ pkgs ? import <nixpkgs> {}}:
+pkgs.mkShell {
+  buildInputs = with pkgs; with ocamlPackages; [
+    ocaml
+    findlib # Required to find the rest of the packages
+    dune_2
```

```suggestion
    ocaml
    findlib # Required to find the rest of the packages
    dune_2
```
This should be added to `nativeBuildInputs`.